pkg/controller/controller.go

```diff
@@ -2,17 +2,26 @@ package controller
 
 import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"github.com/maistra/istio-operator/pkg/controller/common"
 )
 
 // AddToManagerFuncs is a list of functions to add all Controllers to the Manager
 var AddToManagerFuncs []func(manager.Manager) error
 
-// AddToManager adds all Controllers to the Manager
-func AddToManager(m manager.Manager) error {
+
+// Init initializes controller
+func Init(m manager.Manager) error {
+	common.InitCNIStatus(m)
```

Comment:
I think we should leave this function named `AddToManager()` and move this `InitCNIStatus()` code into the initialization of the smcp controller's `Add()` function.